pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -105,6 +105,17 @@ func (c *Client) OperatorExistsInCluster(name, namespace string) bool {
 	return true
 }
 
+// OperatorVersionExistsInCluster checks if a given OperatorVersion object is installed on the current k8s cluster
+func (c *Client) OperatorVersionExistsInCluster(name, namespace string) bool {
+	operator, err := c.kudoClientset.KudoV1beta1().OperatorVersions(namespace).Get(name, v1.GetOptions{})
+	if err != nil {
+		clog.V(2).Printf("operatorversion.kudo.dev/%s does not exist\n", name)
```

Comment:
please change the logging to `namespace/name` here and below (we try to consolidate all logging of objects to this format):
```suggestion
		clog.V(2).Printf("operatorversion %s/%s does not exist\n", namespace, name)
```